ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java

```diff
@@ -113,19 +100,19 @@ void cleanUp() {
         if (mVungleBannerAd != null) {
             Log.d(TAG, "Vungle banner adapter cleanUp: destroyAd # " + mVungleBannerAd.hashCode());
             mVungleBannerAd.destroyAd();
```

Comment:
destroyAd method might run into mAdPlayCallback#onError callback and createBanner#cleanUp will block here until onError operation complete.
In onError operation we call cleanUp again from VungleManger#removeActiveBannerAd and set banner null value.
When createBanner#cleanUp return and going on to the removeView operation, the banner ad already set null value, so we should double check.